src/lib.rs

```diff
@@ -68,6 +69,22 @@ impl SerialPort {
         }
     }
 
+    /// Creates a new serial port interface on the given I/O port.
+    ///
+    /// This function is unsafe because the caller must ensure that the given base address
+    /// really points to a serial port device.
+    #[cfg(feature = "stable")]
```

Comment:
Inline with my comment above, I would prefer here:

```suggestion
    #[cfg(not(feature = "nightly"))]
```

We don't really require the stable feature (e.g. the external assembly) for this function. Instead, we want to use it as a fallback when the nightly feature is not enabled.

Reply:
changed